Update README.md with process description improvements. (#23)
penyaskito/dashboard-initiative: README.md

About the initiative
====

Create a new and modern Dashboard based on user needs, and define those needs per role.

The homepage of the initiative is at the Ideas issue queue. See ["Enhance user experience with customizable dashboards"](https://www.drupal.org/project/ideas/issues/3244581)
(was "Modern Dashboard with Role presets") on drupal.org, which provides a lot of context.

> Development happens in a [sandbox project](https://www.drupal.org/sandbox/penyaskito/3327580). Feel free to create or work on issues in the sandbox.

Reviewer: ```suggestion
Development happens in a [sandbox project](https://www.drupal.org/sandbox/penyaskito/3327580). Feel free to create or work on issues in the sandbox.

If you can, also create a PR here applying the patch via composer.json. This helps with:
* Running the tests, as sandboxes don't have CI enabled.
* Get a tugboat link with a site with the patch applied where people can test the fix/improvement.
* Giving appropiate credit here too for your contribution!
```

Reviewer: Improved the description of the process about how we are using github.

About the initiative
====

Create a new and modern Dashboard based on user needs, and define those needs per role.

The homepage of the initiative is at the Ideas issue queue. See ["Enhance user experience with customizable dashboards"](https://www.drupal.org/project/ideas/issues/3244581)
(was "Modern Dashboard with Role presets") on drupal.org, which provides a lot of context.

> Development happens in a [sandbox project](https://www.drupal.org/sandbox/penyaskito/3327580), as we are still working on getting access to
> the [Dashboard namespace](https://www.drupal.org/project/dashboard). Feel free to create issues in the sandbox, as we will move them when we are able to.

Reviewer: ```suggestion
Development happens in a [sandbox project](https://www.drupal.org/sandbox/penyaskito/3327580). Feel free to create or work on issues in the sandbox.
```

We have access to the dashboard namespace now. We decided to not use it.
